### ISSUE TITLE
Bug - hero bar not in an aria landmark region

### ISSUE THREAD

**Issue by tessereth** [AusDTO/gov-au-ui-kit#313]
Currently, our page has the following structure:

```
<body>
  <div class="skip-to">...</div>
  <header role="banner">...</header>
  <div class="hero-med">...</div>
  <main role="main">...</main>
  <footer role="contentinfo">...</footer>
</body>
```

But the hero is not within an aria landmark region, causing one of the the axe-core best-practice accessibility tests to fail. I am confused how to fix this problem. The hero visually looks like it's part of the header but, at least for the medium hero, the `<h1>` is inside the hero implying that it needs to be part of main. Do you all have a suggested solution to this problem?

**Comment by TrebBrennan** [comment]
Awesome pickup @tessereth

There has been some mild talk about the accessibility and tab-order of the hero so this is quite timely.

My opinion (it's just that) is that `Hero` could be moved into `main` but we'll need to move the styling on main to something else if that's the case.

Added as a bug to our backlog.

**Comment by klepas** [comment]
Tracked under `DESIGN-404`.

**Comment by petronbot** [comment]
Closing this issue since the Hero is as yet undocumented in the Design Guide.
